src/event.rs

```diff
@@ -74,9 +74,94 @@ pub enum Event<'a, T: 'static> {
     UserEvent(T),
 
     /// Emitted when the application has been suspended.
+    ///
+    /// # Portability
+    ///
+    /// Not all platforms support the notion of suspending applications, and there may be no
+    /// technical way to guarantee being able to emit a `Suspended` event if the OS has
+    /// no formal application lifecycle (currently only Android and iOS do). For this reason
+    /// Winit does not currently try to emit pseudo `Suspended` events before the application
+    /// quits on platforms without an application lifecycle.
```

Comment:
I'm personally a big fan of (over)eagerly applying intradoc-links, not sure if _all_ of these should be referencing to `Self::Suspended` and `Self::Resumed`? Also below.

Reply:
The updated docs now consistently link to the other event (i.e. the `Suspended` docs have links to `Resumed` and the `Resumed` docs link back to `Suspended`), and neither links back to itself, it just uses back ticks.